pyccl/halos/profiles.py

```diff
@@ -654,8 +650,9 @@ class HaloProfileNFW(HaloProfile):
             truncated at :math:`r = R_\\Delta` (i.e. zero at larger
             radii.
     """
-    __repr_attrs__ = ("cM", "fourier_analytic", "projected_analytic",
-                      "cumul2d_analytic", "truncated", "precision_fftlog",)
+    __repr_attrs__ = __eq_attrs__ = (
+        "cM", "fourier_analytic", "projected_analytic",
+        "cumul2d_analytic", "truncated", "precision_fftlog",)
```

Comment:
I would put cM last or before precision_fftlog, since this is the slowest one to compare

Reply:
Good idea, done.